repos/system_upgrade/el7toel8/actors/ipuworkflowconfig/libraries/library.py

```diff
@@ -37,3 +38,7 @@ def get_os_release(path):
             severity='high',
             flags=['inhibitor'])
         return None
+
+
+def get_target_version():
+    return os.getenv('LEAPP_DEVEL_TARGET_RELEASE', None) or CURRENT_TARGET_VERSION
```

Comment:
Isn't it the same as `os.getenv('LEAPP_DEVEL_TARGET_RELEASE', CURRENT_TARGET_VERSION)`?

Reply:
No it's not the same. If it's an empty environment variable CURRENT_TARGET_VERSION is used in your version its not using it.

Reply:
Ofc, I could go even more paranoid and check for the stripped string, but I didn't want to go that far